x-pack/plugins/global_search_bar/kibana.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "globalSearchBar",
+  "version": "8.0.0",
+  "kibanaVersion": "kibana",
+  "server": false,
+  "ui": true,
+  "requiredPlugins": ["globalSearch", "globalSearchProviders"],
```

Comment:
The `globalSearchProviders` is not required for the search bar. You only need to have `globalSearch` listed there.